src/Runtime/ExecutionSession.cpp

```diff
@@ -41,6 +41,25 @@ ExecutionSession::ExecutionSession(
     errStr << "Cannot load symbol: '" << entryPointName << "'" << std::endl;
     throw std::runtime_error(errStr.str());
   }
+
+  std::string inputSignatureName("omInputSignature");
```

Comment:
My guess is your failure is coming from the code to load these two functions. Are they always needed now? If so, I think you need to add them to the .def file for the tests.

Reply:
added, thanks for the suggestions. I could make it required only when asked for.

Reply:
@sstamenova you were right on the money, adding the functions names to the .def files was what was needed. It now passes the window tests, thanks for the help.

Reply:
Great! I wish there was a good way to automate this, but I'm glad we figured it out quickly